wordpress-mobile/WordPress-iOS
# Add tip to the top of domain search for free sites

Implements #17191. This PR adds a tip to the top of the domains search screen for free sites:

<img src="https://user-images.githubusercontent.com/4780/136543630-46b63b08-e9db-447d-9d3d-27ec01f9a241.png" width="350">

The text will shrink as needed to a maximum of 3 lines, at which point it will truncate. I tested with very long domains and this seems like a reasonable limit. We need to set a line limit otherwise the text won't shrink, and it can wrap oddly if we don't do that.

<img src="https://user-images.githubusercontent.com/4780/136543686-c358277a-03f0-4af9-9509-fb1c705b4eed.png" width="350">

The tip should only appear for free sites with a redirect.

**To test**

* Build and run
* Navigate to Domains search for a free site. You should see the tip.
* Navigate to Domains search for a site with a premium plan and an unused domain credit. You should not see the tip.

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
